### DISCUSSION TITLE
What about Safety?

### DISCUSSION THREAD

**Discussion by triggerfx** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#531] [General]
Does this mod in any way decrease the safety of the device? Does this modification affect the safety measures taken by the manufacturer, or not?
Is it possible that the modification will cause someone to get an electric shock?

**Comment by visualapproach** [comment]
Modifying things will always bring the risk of damage or injury. Hence the disclaimer on the front page of this project. It's up to the person making the mod to make sure it's safe for himself and others if they decide to proceed with this educational project. That said;
As long as you don't have mains connected while poking around inside the pump the risk for electrical shock is low if you don't mess with the mains stuff.
There is safety functions trying to prevent freezing and overtemp but since it's a computer program they can fail on cpu crash etc.
So if in doubt, don't do this. If you do, it's on you.

**Reply by triggerfx** [reply]
I didn't mean an electric shock while installing it, but while being in the hot tub. But because there is no intervention in the high-voltage area / power supply, that shouldn't normally happen, right?

**Reply by visualapproach** [reply]
You are right. I know it's a boring answer, but I really don't want to be held responsible for what people might do to hurt themselves, so I have to be clear about these things. I will never say "it's safe". I wasn't worried when my family took a swim, but I installed it myself and thought I knew what I was doing.
